data/layout/telescope_positions-North-4LST.ecsv

```diff
@@ -51,8 +46,8 @@
 #       unit: *id001
 #       value: 3.0
 # schema: astropy-2.0
-telescope_name pos_x pos_y pos_z lat lon utm_east utm_north alt
-LST-01 57.5 57.5 57.5 28.762676802629773 -17.89215977351343 217597.94544395315 3185124.2044917224 2142.0
-LST-02 -57.5 -57.5 -57.5 28.761639194765124 -17.892159767691023 217595.1496128795 3185009.1712657204 2142.0
-LST-03 57.5 57.5 57.5 28.762676802629773 -17.89098222648657 217712.9786216382 3185121.4086168357 2142.0
-LST-04 -57.5 -57.5 -57.5 28.761639194765124 -17.890982232308975 217710.1827950689 3185006.3754829657 2142.0
+telescope_name      pos_x      pos_y      pos_z
+LST-01               57.5       57.5      57.5
+LST-02              -57.5      -57.5     -57.5
```

Comment:
Negative z position? I don't think CORSIKA can handle this.

Reply:
It seems it can. I searched in the user guide and there is no mention for limitation in x, y or z. The limitations are in the ID and the radius, which have to be positive. 

I also tested it myself by running manually corsika for one telescope with negative z and it ran smoothly

Reply:
I answered this point in Mattermost, but either way we shouldn't have negative `z`. The lowest telescope should always define the site altitude.

Reply:
As we discussed, I replaced the z values for positive values. it should be fine now